prod/commonvars.hcl

```
@@ -14,4 +14,12 @@ locals {
 
   service_api_url = "http://api-internal.io.italia.it/"
 
+  # Switch limit date for email opt out mode. This value should be used by functions that need to discriminate
+  # how to check isInboxEnabled property on IO profiles, since we have to disable email notifications for default
+  # for all profiles that have been updated before this date. This date should coincide with new IO App's release date
+  # 1625781600000 value refers to 2021-07-09T00:00:00Z
+  opt_out_email_switch_date = 1625781600000
```

```suggestion
  fnapp_opt_out_email_switch_date = 1625781600000
```
maybe it's better to add `fnapp_` prefix in common global vars. Appsetting in function app can be without prefix